### PR TITLE
Add PRR shadows

### PR DESCRIPTION
**Pull request by johnbelamaric**
A number of folks have volunteered to shadow for PRR this cycle. Adding them here so they can access the project board.

### CODE REVIEW

File: config/kubernetes/sig-architecture/teams.yaml [hunk 1]
```diff
@@ -82,8 +82,15 @@ teams:
...
+        - ehashman      # approver
+        - johnbelamaric # approver
+        - wojtek-t      # approver
+        - jeremyrickard # shadow
+        - tallclair     # shadow
+        - bentheelder   # shadow
+        - cici37        # shadow
+        - jpbetz        # shadow
+        - spiffxp       # shadow
+        - andrewsykim   # shadow
```
Comment (palnabarun) [inline config/kubernetes/sig-architecture/teams.yaml:95]: nit: can you please sort the list?

**Review by johnbelamaric — COMMENTED** [review]
Sort team list

**Pull request merged**